packages/next/test/unit/fixtures/01-normalize-paths/index.test.js

```diff
@@ -1,7 +1,7 @@
 const fs = require('fs-extra');
 const ms = require('ms');
 const path = require('path');
-const { build } = require('../../../../src');
+const { build } = require('../../../../dist');
```

Comment:
Was this change intended?

Reply:
Yes. I don't know why it worked before, but if I leave it as `src`, the tests fail with:

```
ENOENT: no such file or directory, open 'C:\Users\chris\projects\vercel\packages\next\src\server-launcher.js'
```

So, I changed it to `dist` and it works as long as you `yarn run build` before running the test. Do you have any idea why?

Reply:
Probably because this is a `.test.js` instead of `.test.ts`

Reply:
Just for fun, I renamed the test to `.ts`, changed the `require()`s to imports, changed `dist` to `src`, and it still blows up with the same "ENOENT: no such file or directory" error above.

The code that references `server-launcher.js` is in `server-build.ts`:

```
    const launcherData = await fs.readFile(
      path.join(__dirname, 'server-launcher.js'),
      'utf8'
    );
```

`server-launcher.js` will only ever exist in the `dist` directory, so it doesn't make any sense why it apparently worked before. In any case, I think referencing `dist` is required.

Reply:
I prefer `dist` anyway. Best to test the code we distribute because thats the code that users run.